test/Conversion/GPUXToLLVM/memset-to-gpu-runtime.test

```diff
@@ -0,0 +1,22 @@
+// RUN: imex-opt --convert-gpux-to-llvm %s | FileCheck %s
```

Comment:
Is this a place holder ? there is no memset and the test looks the same as wait-to-gpu-runtime.test